environments/icds-cas/public.yml

```diff
@@ -145,6 +145,9 @@ couchdb_cluster_settings:
   w: 2
   n: 2
 
+couchdb_compaction_settings:
+  _default: '[{db_fragmentation, "10%"}, {view_fragmentation, "10%"}, {from, "17:00"}, {to, "05:00"}, {strict_window, true}]'
```

Comment:
is 17:00 and 5:00 will respect UTC or IST?

Reply:
Machine time, so it should be UTC (22:00 to 10:00 IST).